src/gradle.properties

```diff
@@ -2,3 +2,5 @@
 // We need to enable access, so that we can refer to external schemas when
 // generating classes with JAXB.
 systemProp.javax.xml.accessExternalSchema=all
+xroadVersion=6.20.0
+xroadBuildType=0
```

Comment:
Maybe "snapshot" and "release" would be clearer. Or the version classifier could actually be an arbitrary string ("beta", "rc1",...).